item.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+
+public class Item
```

Comment:
Rename DBModel Classes to indecate that they are db models. E.G.

`ItemDbModel`